File: tcp/src/main/java/org/lovedev/tcp/MainActivity.java
package org.lovedev.tcp;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        SuClient suClient = new SuClient();
        suClient.init(null);
        suClient.execCMD("sh /sdcard/shine/talk/config_talk.sh");
    }
}

File: mqtt/src/main/java/org/lovedev/mqtt/MainActivity.kt
package org.lovedev.mqtt

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import org.eclipse.paho.client.mqttv3.MqttClient


class MainActivity : AppCompatActivity() {
    private val serverUri = "ws://10.0.1.86:8083"
    private var clientId = "ExampleAndroidClient"
    private val publishTopic = "exampleAndroidPublishTopic"
    lateinit var mqttAndroidClient: MqttClient
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val intent = Intent(this, CustomMQTTService::class.java)
        startService(intent)
    }
}
